plugins/by-name/wrapping/default.nix

```
@@ -18,9 +14,68 @@ lib.nixvim.neovim-plugin.mkNeovimPlugin {
       By default, wrapping.nvim will output a message to the
       command line when the hard or soft mode is set.
     '';
+
+    create_commands = defaultNullOpts.mkBool true ''
+      If true, the plugin will create the following
+      commands to set/override a wrapping mode in case
+      it is not autodetected correctly:
+
+      * HardWrapMode
```

```suggestion
      * `HardWrapMode`
```

Where we have symbol names, mappings, or code snippets it's best to use `` `backticks` `` to render a `<code>` span.